doc/transactions.md

```diff
@@ -0,0 +1,91 @@
+Transactions
+============
+Transactions is a feature that comes from SOFT, which allows to easy manage arbitrary long series of immutable (frozen) instances, while ensuring provenance.
```

Comment:
```suggestion
============

Transactions is a feature that comes from SOFT, which allows to easy manage arbitrary long series of immutable (frozen) instances, while ensuring provenance.
```

Reply:
Do you have a reference to the part in SOFT where this comes from? If not, that's fine.